http://lists.beasts.org/pipermail/iftop-users/2014-March/000414.html

From: Hans Fugal <hans@fugal.net>
Date: Fri, 7 Mar 2014 13:22:18 -0800
Subject: [PATCH 2/3] scale[] up to tbit

Extend the scale[] array up to terabit. 10gbit is not uncommon,
100gbit 40 and 100 gbit are coming, 400 gbit and terabit are future
possibilities.
---
 ui.c | 17 ++++++++++-------
 1 file changed, 10 insertions(+), 7 deletions(-)

diff --git a/ui.c b/ui.c
index 8a3b9d0..d1500ad 100644
--- a/ui.c
+++ b/ui.c
@@ -74,13 +74,16 @@ static struct {
     double max;
     int interval;
 } scale[] = {
-        {      64000,     10 },     /* 64 kbit/s */
-        {     128000,     10 },
-        {     256000,     10 },
-        {    1000000,     10 },     /* 1 Mbit/s */
-        {   10000000,     10 },     
-        {  100000000,    100 },
-        { 1000000000,    100 }      /* 1 Gbit/s */
+        {         64000,     10 },     /* 64 kbit/s */
+        {        128000,     10 },
+        {        256000,     10 },
+        {       1000000,     10 },     /* 1 Mbit/s */
+        {      10000000,     10 },
+        {     100000000,    100 },
+        {    1000000000,    100 },     /* 1 Gbit/s */
+        {   10000000000,    100 },
+        {  100000000000,    100 },
+        { 1000000000000,    100 },     /* 1 Tbit/s */
     };
 static int rateidx = 0, wantbiggerrate;
 
